definitions/variable/emissions/tag_land-removal-options.yaml

```diff
@@ -0,0 +1,16 @@
+- Land Removal Option:
+    - Agroforestry:
+        description: agroforestry
+    - Afforestation:
+        description: afforestation
+    - Biochar:
+        description: conversion of biomass to recalcitrant carbon via pyrolysis and its
+          subsequent application to croplands to increase soil carbon stocks
+    - Soil Carbon Management:
+        description: soil carbon management techniques
+    - Soil Carbon Management|Cropland:
+        description: soil carbon management techniques on cropland
+    - Soil Carbon Management|Grassland:
+        description: soil carbon management techniques on grassland
```

Comment:
perhaps call back to upper level variable where more context will be provided: "See `Soil Carbon Management` for more information"